source/beamline/BioXAS/BioXASSSRLMonochromatorMaskState.cpp

```diff
@@ -0,0 +1,100 @@
+#include "BioXASSSRLMonochromatorMaskState.h"
+#include "actions3/AMActionSupport.h"
+#include "actions3/AMListAction3.h"
+
+BioXASSSRLMonochromatorMaskState::BioXASSSRLMonochromatorMaskState(const QString &name, QObject *parent) :
+	AMSingleEnumeratedControl(name, "", parent)
```

Comment:
Since this has two controls, shouldn't this be a descendent of AMEnumeratedControl rather than AMSingleEnumeratedControl?